ddtrace/propagation/http.py

```diff
@@ -60,6 +70,33 @@ def parent_call():
         if span_context.dd_origin is not None:
             headers[HTTP_HEADER_ORIGIN] = str(span_context.dd_origin)
 
+        # Only propagate tags that start with `_dd.p.`
+        tags_to_encode = {}  # type: Dict[str, str]
```

Comment:
Being that we support unicode tag values, we should include some cases in tests below.

Reply:
I can add 👍🏻, but we also don't support propagating arbitrary tags, only specific internal tags that must be ascii

Reply:
will be good to have so we can codify the expected error handling

Reply:
handled, I wasn't actually enforcing ascii only in `tagset` encoding so I added that as well here.